main.go

```diff
@@ -11,9 +14,11 @@ import (
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/rate_limit/v2"
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/redis_proxy/v2"
 	_ "github.com/envoyproxy/go-control-plane/envoy/config/filter/network/tcp_proxy/v2"
+	"github.com/ghodss/yaml"
```

Comment:
Why use a different yaml library here compared to everywhere else?

Reply:
Changed to use only one lib everywhere. (was not intentional, vscode tooling pulled this yaml lib instead)